package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "day1",
+  "version": "1.0.0",
+  "description": "day 1 - advent of code",
+  "main": "index.js",
+  "dependencies": {
+    "jest": "^20.0.4"
```

Comment:
Is common to put tests library in `devDependencies` node, because tests will not run on production environment.